src/content/en/tools/chrome-devtools/javascript/step-code.md

```diff
@@ -204,9 +229,9 @@ To blackbox a file:
 
 3. Click **Add pattern**.
 
-4. In the **Pattern** textfield enter the filename pattern that you wish to 
-   exclude from your call stack. DevTools excludes any scripts that match the 
-   pattern. 
+4. In the **Pattern** textfield enter the filename pattern that you wish to
```

Comment:
Thanks, will remember going forward. This is a deprecated page so I'm not going to bother with it.